.eslintrc.json

```diff
@@ -28,6 +28,15 @@
   "rules": {
     "react/react-in-jsx-scope": "off",
     "react/prop-types": "off",
+    "jsx-a11y/label-has-associated-control": [
+      2,
+      {
+        "labelComponents": ["CustomInputLabel"],
+        "labelAttributes": ["label"],
```

Comment:
_"`labelAttributes` is a list of attributes to check on the label component and its children for a label."_

Since we don't have those custom label components, this row isn't needed either.